resources/assets/js/vue-i18n-locales.generated.js

```diff
@@ -0,0 +1,1539 @@
+export default {
```

Comment:
Ik zou dit ding dus nooit in version control zetten, maar genereren on the fly. Dan weet je altijd dat je een up-to-date versie hebt 🙂

Reply:
We kunnen hem ignoren en opnemen in het InstallCommand? Of ook een RunCommand? Genoeg mogelijkheden